### PR TITLE
add more clarification on the perps documentation.

### PR DESCRIPTION
**Pull request by siong1987**

### CODE REVIEW

File: labs/2-perpetual-exchange/3-how-it-works.md [hunk 1]
```diff
@@ -55,22 +56,22 @@ custody's token to the LP's desired token.
...
 
 For example, suppose SOL is at 50% utilization and assuming an hourly funding
 rate of 0.01%. Then a trader with a long position SOL-USD of size 1000 USD will
 accumulate funding fees at a rate of 0.05 USD per hour.
 
-### Auto closing positions that exceed maximum leverage
+### Auto Closing Positions that Exceed Maximum Leverage
 
-The maximum allowed leverage is 100x.
+The maximum allowed leverage is 250x.
```
Comment (siong1987) [inline labs/2-perpetual-exchange/3-how-it-works.md:72]: @hjchan is this correct?

File: labs/2-perpetual-exchange/2-jlp-pool.md [hunk 2]
```diff
@@ -85,12 +85,15 @@ Apart from the withdrawal/deposit fee mechanisms, JLP pool is also integrated in
 | Opening a Position | 10 BPS |
 | Closing a Position | 10 BPS |
 | Swap Fee | Between 0 BPS to 200 BPS depending on pool weightage |
+| Borrow Rate | 1 BPS/hour |
```
Comment (hjchan) [inline labs/2-perpetual-exchange/2-jlp-pool.md:88]: this need to times utilisation rate also

File: labs/2-perpetual-exchange/3-how-it-works.md (hunk 1 above)
Comment (hjchan) [reply labs/2-perpetual-exchange/3-how-it-works.md:72]: 200x

File: labs/2-perpetual-exchange/3-how-it-works.md [hunk 3]
```diff
@@ -55,22 +56,22 @@ custody's token to the LP's desired token.
...
 
 Positions where the trader's collateral less fees and less unrealized losses is
-less than 1% of the position size are automatically closed.
+less than 0.4% of the position size are automatically closed.
```
Comment (hjchan) [inline labs/2-perpetual-exchange/3-how-it-works.md:75]: 0.5%

**Review by hjchan — APPROVED** [review]

**Pull request merged**